creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -134,13 +134,42 @@ const retrieveClockValueForUserFromReplica = async (replica, wallet) => {
 
 /**
  * Returns an object that can be returned from any state machine job to record a histogram metric being observed.
- * Example: to call histogram.observe('response_time', { code: '200' }, 1000), you would call this function with:
+ * Example: to call responseTimeHistogram.observe({ code: '200' }, 1000), you would call this function with:
  * makeHistogramToRecord('response_time', 1000, { code: '200' })
  * @param {string} metricName the name of the metric from prometheus.constants
  * @param {number} metricValue the value to observe
  * @param {string} [metricLabels] the optional mapping of metric label name => metric label value
  */
 const makeHistogramToRecord = (metricName, metricValue, metricLabels = {}) => {
+  return makeMetricToRecord('HISTOGRAM', metricName, metricValue, metricLabels)
```

Comment:
can you move these types to constants file

Reply:
its a bit diff than metricType of histogram vs gauge, since this is metricRecordType or smth, histogram vs gauge vs gauge_inc

Reply:
good callout -- changed